llvm/lib/Target/RISCV/RISCVInstrInfoVPseudos.td

```diff
@@ -2060,12 +2059,12 @@ multiclass VPseudoVIOTA_M {
   defvar constraint = "@earlyclobber $rd";
   foreach m = MxList in {
     defvar mx = m.MX;
-    let VLMul = m.value in {
+    let VLMul = m.value, ActiveElementsAffectResult = 1 in {
       def "_" # mx : VPseudoUnaryNoMask<m.vrclass, VR, constraint>,
```

Comment:
We don't set `ActiveElementsAffectResult` for unmasked before?

Reply:
That's part of why it was confusing, unmasked pseudos don't have RISCVMaskedPseudo entries. 

So to check if an unmasked pseudo had `ActiveElementsAffectResult`, you had to lookup it's masked pseudo entry first, and then check if it's masked pseudo had `ActiveElementsAffectResult`.